modules/ROOT/pages/guided-setup.adoc

```diff
@@ -81,9 +81,9 @@ Map community user to Anypoint role:
 +
 If you work with different business groups, you must create a new external group for each business group in your organization.
 
-== Next Steps
+== Profile Permissions
```

Comment:
Isn't this really a task about updating permissions?  Maybe some of the topics above should be recast as tasks as well.

Reply:
True, this is a task page. Updated so all sections are clearly tasks.